target/riscv/cpu.c

```diff
@@ -95,6 +95,8 @@ const char * const riscv_excp_names[] = {
     "guest_load_page_fault",
     "reserved",
     "guest_store_page_fault",
+    [RISCV_EXCP_LOAD_CAP_PAGE_FAULT] = "load_cap_page_fault",
+    [RISCV_EXCP_STORE_AMO_CAP_PAGE_FAULT] = "store_cap_page_fault",
```

Comment:
Description lacks _amo, but matches the other STORE_AMO faults so this is fine.

Reply:
Yes, that was my thinking as well when I went back and renamed the constants to add AMO.